.circleci/config.yml

```diff
@@ -117,38 +125,41 @@ commands:
             ${SUDO} apt-get install -y unixodbc-dev
 
       - run: &activate_virtualenv
-          name: Run make pipenv-lock and create virtualenv
+          name: Create virtualenv
           command: |
             echo 'Checking for virtualenv'
             if [ ! -d venv ] ; then
+              rm -rf ~/.cache/pipenv
```

Comment:
Perhaps we shouldn't cache the entire `~/.cache` folder in the Yarn task?  
Then we won't have to delete the `pipenv` cache here

Reply:
Whoops, yes good call. I'm going to leave the cache alone.